### PR TITLE
created desktop version

### PR DESCRIPTION
**Pull request by danielmatama**
edited HTML and CSS for desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
...
+
+      <div class="end">
+      <ul class="social-icons">
+        <li class="icon"><a href="#"><img src="images/twitter.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/medium.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/github.svg" alt=""></a></li>
+        <li class="icon"><a href="#"><img src="images/angel.svg" alt=""></a></li>
+      </ul>
+        </div>
```
Comment (brainconnect93) [inline index.html:278]: - [ ] You have done a great job so far with your desktop design 👏 👏. But there are some little differences between your design and the design in the Figma template you have chosen. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- Your project Design👇 (Please pay attention to my highlight below in the contact section of your project):

1.  In the `Desktop` version of this particular template you need to add one more `input` tag to your form that will hold your `Last Name`.
2. Kindly check the `background image` of your design as it is coming under the `form` and the `text`. Kindly check the Figma design and make adjustments.

Your design 👇 

dan1

Figma Design 👇 

cont real

File: index.html [hunk 2]
```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
...
+      <div class="about-me">
+        <div class="about-me-text">
+        <h1 class="intro">About Me</h1>
+        <p class="about-text"> I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+        </div>
+        <div class="text-center">
           <button type="button" class="btn btn-success resume-button">Get My Resume</button>
+        </div>
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you follow the Figma template design as much as possible, As currently the `About section` of your project is missing one of the `background image` as we have it in the Figma template and the second one is overflowing with the `text`, Kindly check the Figma template to rectify it.

Your Design 👇 

dan abt

Figma Design 👇 

abt real

File: index.html [hunk 3]
```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
...
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
```
Comment (brainconnect93) [inline index.html]: - [ ] You have really made a great effort in working on the `work` aspect of your project but it seems the styling is lacking some consistency with the Figma template. Your design looks different from what we have in the Figma template
- [ ] Also please attach a `border radius` to the `Ruby on rails`, `CSS`, `JavaScript`, and `HTML` as seen in the Figma design for both ```mobile``` and ```desktop``` versions.

Figma Design 👇 

work card

Your Design 👇 

dan work

File: index.html [hunk 4]
```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
...
+          <img src="images/sphere.svg" class="sp-image" alt="">
+        </div>
+        <h2 class="card-text2">Languages</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc2">Javascript</a></li>
+          <li><a href="#" class="anc2">Ruby</a></li>
+          <li><a href="#" class="anc2">Html</a></li>
+          <li><a href="#" class="anc2">CSS</a></li>
+        </ul>
+
```
Comment (brainconnect93) [inline index.html]: - [ ] Kindly stick to the Figma template as much as possible for a better experience. Currently, on the `mobile version` of your project, the `Languages`, `Frameworks`, and `Skills` cards do not look like what we have in the Figma template you worked on. Kindly fix that and let them look alike.

- Also please attach a `border` and `border radius` around the `Ruby on rails`, `CSS`, `JavaScript`, and `HTML` as seen in the Figma design for the mobile version.

Figma Design 👇 

mob rel

Your design 👇 

dan mob

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @danielmatama, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (danielmatama) [reply index.html:278]: we were instructed to ignore the additional input field for name and use 1 field on desktop, like the one on mobile

**Comment by danielmatama** [comment]
please note that we were instructed to use 1 input field for the name instead of 2, for both mobile and desktop versions

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅

## Hi @danielmatama 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 

## Optional suggestions:
- [ ] Please consider following best practices when writing commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" 👍
- [ ] Please consider using `flexbox` to center your elements instead of margins or paddings, flexbox is responsive and it calculates the center of every width 👍 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**